lib/Default/SmrAlliance.class.php

```diff
@@ -244,6 +244,9 @@ public function setImageURL($url) {
 		$this->imgSrc = htmlspecialchars($url);
 	}
 
+	/**
+	 * Get the total credits in the alliance bank account.
```

Comment:
It might make more sense to just rename the function so it's clear wherever it's called without even needing to look at the docs, eg getAccountBalance?

Reply:
It definitely would! I just didn't want to make more work for the author of #829 (who is, surprisingly, not me).

Reply:
For the record I don't mind doing extra work in the name of making things orderly and sensible.

Reply:
Then your wish is my command! I've renamed `SmrAlliance::getAccount` to `SmrAlliance::getBank` (for consistency with SmrPlayer methods).